src/improvements/tasks/eth/001-opcm-upgrade-v200/VALIDATION.md

```diff
@@ -0,0 +1,269 @@
+```
+
+### Decode Multicall3DelegateCall calldata:
+```bash
+cast calldata-decode 'aggregate3((address,bool,bytes)[])' <0x82ad56cb...>
+
+[
+    (
+        0x026b2F158255Beac46c1E7c6b8BbF29A4b6A7B76,
+        false, 
+        0xff2dd5a10000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000100000000000000000000000073a79fab69143498ed3712e519a88a918e1f40720000000000000000000000000475cbcaebd9ce8afa5025828d5b98dfb67e059e039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d9
+    )
+]
+```
+
+1. First tuple (Call3 struct for Multicall3DelegateCall)
+    - `target`: [0x026b2F158255Beac46c1E7c6b8BbF29A4b6A7B76](https://github.com/ethereum-optimism/superchain-registry/blob/d4bb112dc979fd43ac92252c549d3ed7c4d0eb57/validation/standard/standard-versions-mainnet.toml#L21) - Mainnet OPContractsManager v2.0.0
+    - `allowFailure`: false
+    - `callData`: `0xff2dd5a1...` See below for decoding.
+        - Command to encode: `cast calldata 'aggregate3((address,bool,bytes)[])' "[(0x026b2F158255Beac46c1E7c6b8BbF29A4b6A7B76,false,0xff2dd5a10000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000100000000000000000000000073a79fab69143498ed3712e519a88a918e1f40720000000000000000000000000475cbcaebd9ce8afa5025828d5b98dfb67e059e039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d9)]"`
+
+### Decode upgrade calldata
+
+```bash
+cast calldata-decode 'upgrade((address,address,bytes32)[])' <0xff2dd5a1...>
+
+[
+    (
+        0x73a79Fab69143498Ed3712e519A88a918e1f4072,
+        0x0475cBCAebd9CE8AfA5025828d5b98DFb67E059E,
+        0x039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d9
+    )
+]
+```
+1. First tuple (Base):
+    - SystemConfigProxy: [0x73a79Fab69143498Ed3712e519A88a918e1f4072](https://github.com/ethereum-optimism/superchain-registry/blob/d4bb112dc979fd43ac92252c549d3ed7c4d0eb57/superchain/configs/mainnet/base.toml#L59)
+    - ProxyAdmin: [0x0475cBCAebd9CE8AfA5025828d5b98DFb67E059E](https://github.com/ethereum-optimism/superchain-registry/blob/d4bb112dc979fd43ac92252c549d3ed7c4d0eb57/superchain/configs/mainnet/base.toml#L60)
+    - AbsolutePrestate: [0x039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d9](https://github.com/ethereum-optimism/superchain-registry/blob/1a5d7a208cea9b0ea175df1fe71bdc4da7f4c04c/validation/standard/standard-prestates.toml#L22)
+    - Command to encode: `cast calldata 'upgrade((address,address,bytes32)[])' "[(0x73a79Fab69143498Ed3712e519A88a918e1f4072,0x0475cBCAebd9CE8AfA5025828d5b98DFb67E059E, 0x039facea52b20c605c05efb0a33560a92de7074218998f75bcdf61e8989cb5d9)]"`
+
+## Tenderly State Changes
+
+[Link](https://dashboard.tenderly.co/oplabs/op-mainnet/simulator/4e0bc0ca-1b42-4ffe-bbe8-b648d9ea4971)
+
+## Auto Generated Task State Changes
+
+```bash
+----------------- Task State Changes -------------------
+  
+----- DecodedStateDiff[0] -----
```

Comment:
You'll need to add in links to addresses like we've done for other runbooks. This is time consuming but needs to be done for reviewers of this PR and signers. Example here: https://github.com/ethereum-optimism/superchain-ops/blob/e5a7f7b659f5925b6b2377ddc489e20ced47adfd/src/improvements/tasks/eth/002-opcm-upgrade-v200/VALIDATION.md